packages/apex-node/test/execute/executeService.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Apex Execute Tests', async () => {
   beforeEach(async () => {
     sandboxStub = createSandbox();
 
-    $$.setConfigStubContents('GlobalInfo', {
+    $$.setConfigStubContents('AuthInfoConfig', {
```

Comment:
You can remove all of these. This is essentially what `$$.stubAuths()` is doing for you